symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -332,6 +338,7 @@ public function appendAttachment($file)
                 'file' => $file,
                 'filename' => null,
                 'charset' => null,
+                'cid' => false,
```

Comment:
I think we don't need to add `cid` here, because it has a default value of `null` anyway in `EmailGateway::contentInfoString` and `EmailGateway::contentInfoArray` alike.

Reply:
For PHP to be happy, you are right. I think we might add some isset() checks to make PHP super happy. And it felt like this was like documentation also. WDYT ?

Reply:
If you want to explicitely set params to their default values you should add `'disposition' => 'attachment'` as well. :-)

Honestly, I don't get your idea here. To me that's not documentation. The comment sounds like "this is old rubbish": https://github.com/symphonycms/symphony-2/blob/3da6eeb9a87096b8ece5f8247830759929f3b107/symphony/lib/toolkit/class.emailgateway.php#L330, so for Symphony 3.0 you might even decide to no longer support this legacy format for setting attachments. I wouldn't oppose removing this block altogether.

Reply:
> you might even decide to no longer support this legacy format for setting attachments.

Sorry, I've misunderstood what you meant. Yeah let's get rid of the old format !!

I'll remove the "doc" values ;)